### DISCUSSION TITLE
Any options for music control?

### DISCUSSION THREAD

**Discussion by toddsay** [agronick/aa-torque#103] [Ideas]
This app looks awesome, great work! For me, the main use of Android Auto is playing music, but it would be great to be able to add gauges like this too! But I saw you had a feature to show media images so it got me wondering...

Is there a possibility of actually controlling the current music player? If I could see the current track and artist, and maybe have buttons for next/previous/pause, it seems like this could also serve as the music player/controller (and just to dream big, also the track time and progress). I don't know if Android gives you access to that level of control. But if it did, I could envision some music details at the top/bottom (or even as a replacement for one of the gauges).

Is my dream a possible enhancement? And if not, I'm curious how others handle playing music while running this app.

**Comment by agronick** [comment]
It could be added pretty easily but a lot of devices already show the music controls on the bottom. It is only the widescreen setup that doesn't show the media on the screen. I might add the metadata information at the top for people like myself that have the widescreen. Controlling the media is already doable through hardware buttons or steering wheel controls.

**Reply by toddsay** [reply]
I didn't realize that (mine is a widescreen too). It's true I have hardware buttons for next/prev (over on the right side of the screen; it works but is a bit of a reach), and I have a steering wheel volume control with up/down navigation, but nothing that does play/pause.
